packages/snaps-rpc-methods/src/permitted/scheduleBackgroundEvent.ts

```diff
@@ -56,31 +54,13 @@ export const scheduleBackgroundEventHandler: PermittedHandlerExport<
   hookNames,
 };
 
-const offsetRegex = /Z|([+-]\d{2}:?\d{2})$/u;
-
 const ScheduleBackgroundEventParametersWithDateStruct = object({
-  date: refine(string(), 'date', (val) => {
```

Comment:
Wondering if changes like this should be in a follow-up PR. Do you think it would be possible to reduce the scope for the PR?

Reply:
Let me split up the PR to help with this

Reply:
https://github.com/MetaMask/snaps/pull/3287